src/styles/themes/default.ts

```diff
@@ -82,7 +82,7 @@ const darkTheme = {
     starDefaultBG: 'rgb(254, 228, 94)',
     loungeAccessOverlay: colors.blue800,
     mapAttributionText: colors.black,
-    PAGE_BACKGROUND_COLORS: {},
+    PAGE_BACKGROUND_COLORS: {} as PageBackgroundColors,
```

Comment:
How about moving colors into the object so `satisfies ThemeBase` works as expected?

```suggestion
    PAGE_BACKGROUND_COLORS: {
        [SCREENS.HOME]: colors.darkAppBackground,
        [SCREENS.SAVE_THE_WORLD.ROOT]: colors.tangerine800,
        [SCREENS.SETTINGS.PREFERENCES]: colors.blue500,
        [SCREENS.SETTINGS.WORKSPACES]: colors.pink800,
        [SCREENS.SETTINGS.SECURITY]: colors.ice500,
        [SCREENS.SETTINGS.STATUS]: colors.green700,
        [SCREENS.SETTINGS.ROOT]: colors.darkAppBackground,
    },
```

Reply:
and same for light theme

Reply:
@kubabutkiewicz Actually you can wait for this [PR](https://github.com/Expensify/App/pull/24178) to be merged first